src/hooks/useWs/index.tsx

```diff
@@ -19,10 +19,23 @@ const useWS = <T extends TSocketEndpointNames>(name: T) => {
   }, []);
 
   const send = useCallback(
-    async (data?: Parameters<typeof apiManager.augmentedSend<T>>[1]) => {
+    async (data?: Parameters<typeof apiManager.augmentedSend<T>>[0]) => {
+      let payload = data;
+
+      if (name) {
+        if (payload === undefined) {
+          payload = { [name]: 1, ...payload };
+        } else if (name == 'api_token') {
+          payload = { [name]: 1, ...payload };
```

Comment:
How is this different from if block in line 27?

Reply:
done